.travis.yml

```diff
@@ -2,7 +2,7 @@
 sudo: false
 language: ruby
 cache: bundler
-bundler_args: --without development
+bundler_args: --without system_tests development
```

Comment:
Does this need to be done via modulesync and https://github.com/voxpupuli/puppet-corosync/blob/master/.sync.yml ?